### PR TITLE
Clarifying the priority of constituencies

### PR DESCRIPTION
**Pull request by hadleybeeman**
Amended the priority of constituencies, to be clearer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -191,6 +191,7 @@ <h2>Principles</h2>
 we will consider what harm it could do to society or groups,
 especially vulnerable people.
 We will prioritize potential benefits for users over potential benefits to web developers,
+user agents, advertisers or others in the ecosystem,
```
Comment (dbaron) [inline index.html]: maybe also mention "content providers" or something like it (perhaps either before or after "user agents"?)

Comment (torgo) [reply index.html]: Should we clarify "end users" as well?

Comment (hadleybeeman) [reply index.html:194]:
```suggestion
We will prioritise potential benefits for end users over potential benefits to web developers, content providers, user agents, advertizers or others in the ecosystem,
```

Comment (hadleybeeman) [reply index.html]: How's this: "We will prioritise potential benefits for end users over potential benefits to web developers, content providers, user agents, advertizers or others in the ecosystem,"
?

Comment (dbaron) [reply index.html]: Sounds good to me, modulo leaving the spelling as it was before (prioritize, advertisers), at least if we're hoping to keep the document in US English.

Comment (rhiaro) [inline index.html]:
```suggestion
We will prioritize potential benefits for end users over potential benefits to web developers,
content providers, user agents, advertisers or others in the ecosystem,
```

**Pull request merged**